hooks/pre-commit

```diff
@@ -9,14 +9,13 @@ import os
 import re
 import sys
 import subprocess
-import urllib.request
+import requests
 
 pattern = re.compile(r'[."]version[\'"\s":=]+(\d+\.\d+.\d+)')
 version_dict = {}
-version_file = urllib.request.urlopen(
-    "https://raw.githubusercontent.com/weecology/retriever/master/version.txt")
-version_file.readline()
-
+version_file = requests.get(
+    "https://raw.githubusercontent.com/weecology/retriever/master/version.txt").text
+version_file = version_file.splitlines()[1:]
 
 def to_string(value):
```

Comment:
If we are taking out to_strings, curious if we need it in other places too?
From a naive view I think ` requests.get().text` handles that. I am not sure.
Otherwise it feels like this `to_sting` would have been useful if we were doing  `requests.get().content`